addons/website_quote/controllers/main.py

```diff
@@ -89,7 +89,7 @@ def view(self, order_id, pdf=None, token=None, message=False, **post):
         }
 
         if order_sudo.require_payment or values['need_payment']:
-            acquirers = request.env['payment.acquirer'].sudo().search([('website_published', '=', True), ('company_id', '=', order_sudo.company_id.id)])
+            acquirers = request.env['payment.acquirer'].sudo().search([('website_published', '=', True), ('company_id', '=', order_sudo.company_id.id),  '|', ('specific_countries', '=', False), ('country_ids', '=', order_sudo.partner_id.country_id.id)])
```

Comment:
I think you should use `in`: `('country_ids', 'in', [order_sudo.partner_id.country_id.id])`



e.g. https://github.com/odoo/odoo/blob/11.0/addons/mail/controllers/bus.py#L27